code/debugpanel.go

```diff
@@ -250,6 +250,12 @@ func (dv *DebugPanel) InitTabs() {
 	})
 }
 
+// PanelIsValid checks if panel is valid for use.
+// for outside functions calling in, e.g., textbuf.
+func (dv *DebugPanel) PanelIsValid() bool {
```

Comment:
Should this be unexported?

Reply:
there is a lot of unexporting that could be done here, but really it doesn't matter as this is not a toolkit and nobody cares.

Reply:
It does matter and it is important for us to maintain code quality. It is obviously less urgent and we are not going to go through and clean everything up now, but you should at least make new changes properly.